pandas/core/series.py

```diff
@@ -435,6 +435,51 @@ def dtypes(self) -> DtypeObj:
 
     @property
     def name(self) -> Label:
+        """
+        Return the name of the Series.
+
+        The name of a Series becomes its index or column name if it is used
+        to form a DataFrame. It is also used whenever displaying the Series
+        using the interpreter, and persists through some transformations.
+
+        Returns
+        -------
+        Label (int, str or other hashable object).
```

Comment:
The format of Returns is the type first (without an ending period), and the description in the next line:
```
Returns
-------
label
    The name of the Series, which is the column name if it's part of a DataFrame.
```

Reply:
Thanks, I've changed the format. I don't see any other reference to custom pandas types like label, does capitalisation matter?